master/buildbot/test/unit/test_reporters_irc.py

```diff
@@ -53,6 +53,14 @@ def makeBot(self, *args, **kwargs):
             args = ('nick', 'pass', ['#ch'], [], [], {})
         return irc.IrcStatusBot(*args, **kwargs)
 
+    def test_groupDescribe(self):
+        b = self.makeBot()
+        b.describe = lambda d, m: evts.append(('n', d, m))
+
+        evts = []
```

Comment:
should `evts` be define *before* it's used?  (as a side note: what prevents calling it `events`?)

Reply:
This added test is a direct copy of the two following tests, with minimal changes as required to test the different method.

Reply:
I see no reason not to improve things.  Blind copy-n-paste is not really helping.  If I have added this test, I'd also add a "cosmetic changes" commit to make things more readable.  It's up to you and other reviewers though...

Reply:
I think it is better to keep consitency within the file, and add style improvments on other PR. This helps the code style and help stuff getting in faster, IMHO. 
@p12tic and @mokibit are currently doing such excelent work in their latest submitted PRs

Reply:
Ok.